app/authentication/authenticator.py

```diff
@@ -110,6 +110,9 @@ def store_session(metadata):
     questionnaire_store = get_questionnaire_store(user_id, user_ik)
     questionnaire_store.metadata = metadata
     questionnaire_store.add_or_update()
+
+    login_user(User(user_id, user_ik))
```

Comment:
I don't think we need this now we're getting values from session_store.

Reply:
Technically we don't, but this is still worth having as it saves a db call.

Reply:
Maybe add a comment just to indicate that the reasoning behind this line is non-functional?